backend/functions/lib/adapters/gateways/storage/ustp-office-data-map.ts

```diff
@@ -0,0 +1,116 @@
+import { UstpOfficeDetails } from '../../../../../../common/src/cams/courts';
+
+//TODO: We should probably put this in Cosmos so we don't have to deal with this locally
+export const USTP_OFFICE_DATA_MAP = new Map<string, UstpOfficeDetails>([
```

Comment:
We should probably add the following in addition to NY and Seattle:

```
  'USTP_CAMS_Region_2_Office_Brooklyn,USTP CAMS Region 2 Office Brooklyn,BR\n' +
  'USTP_CAMS_Region_2_Office_Central_Islip,USTP CAMS Region 2 Office Central Islip,LI\n' +
  'USTP_CAMS_Region_2_Office_Albany,USTP CAMS Region 2 Office Albany,AL\n' +
  'USTP_CAMS_Region_2_Office_Utica,USTP CAMS Region 2 Office Utica,UT\n' +
  'USTP_CAMS_Region_2_Office_Buffalo,USTP CAMS Region 2 Office Buffalo,BU\n' +
  'USTP_CAMS_Region_2_Office_Rochester,USTP CAMS Region 2 Office Rochester,RO\n' +
  'USTP_CAMS_Region_2_Office_New_Haven,USTP CAMS Region 2 Office New Haven,NH\n' +
```

Reply:
At least before we end work on CAMS-283.

Reply:
I've added a goal to our tracker.